Package.swift

```diff
@@ -59,7 +59,7 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "1.5.5"),
         .package(url: "https://github.com/pointfreeco/swift-identified-collections", from: "1.0.0"),
         .package(url: "https://github.com/pointfreeco/swift-macro-testing", from: "0.4.0"),
-        .package(url: "https://github.com/pointfreeco/swift-perception", from: "1.5.0"),
+        .package(url: "https://github.com/pointfreeco/swift-perception", from: "2.0.0"),
```

Comment:
Since we didn't have to make any code changes, what do you think about allowing a range here, like we do for swift-syntax? Then we could freely upgrade or rollback downstream without having to touch workflow again.

something like this?
```suggestion
        .package(url: "https://github.com/pointfreeco/swift-perception", "1.5.0"..<"3.0.0"),
```

Reply:
Yeah I think that makes sense.

I think the only risk is we start depending on newer API and don't remember to bump it back up. Not a big deal, it'd just be like https://github.com/square/workflow-swift/pull/375